flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/stream/StreamExecDeduplicate.java

```diff
@@ -89,6 +91,17 @@
                                     + "but there will be additional overhead."
                                     + "Default is true.");
 
+    @Experimental
+    public static final ConfigOption<Boolean> TABLE_EXEC_DEDUPLICATE_MINIBATCH_COMPACT_CHANGES =
+            ConfigOptions.key("table.exec.deduplicate.mini-batch.compact-changes")
+                    .booleanType()
+                    .defaultValue(false)
+                    .withDescription(
+                            "Set whether send all change log for row-time mini-batch."
+                                    + "If true, Flink will only send the latest change log to downstream like proc-time."
```

Comment:
If true, Flink will compact changes, only send the latest change to downstream. Notes: If the downstream needs the details of versioned data, this optimization cannot be opened.
If false, Flink will send all changes to downstream just like when the mini-batch is not on.